data_inconsistencies.py

```diff
@@ -0,0 +1,54 @@
+import os
+import csv
+
+
+def get_inconsistencies():
+    files = os.listdir('./data/')
+
+    parsed = {}
+    for file in files:
+        parts = os.path.splitext(file)[0].split('_')
+
+        ID = int(parts[0])
+        if ID not in parsed:
+            parsed[ID] = {}
+        element = '_'.join(parts[1:-1])
+        if element:
```

Comment:
You could avoid this variable here and use the length of parts. If there are two parts, it's the short format, if there are more it's the other format.